src/runtime/Module.cpp

```diff
@@ -282,7 +282,7 @@ Instance* Module::instantiate(ExecutionState& state, const ExternVector& imports
                          &data);
             }
 
-            if (UNLIKELY(elem->tableIndex() >= numberOfTableTypes() || index >= instance->m_tables[elem->tableIndex()]->size() || index + elem->functionIndex().size() > instance->m_tables[elem->tableIndex()]->size())) {
+            if (UNLIKELY(elem->tableIndex() >= numberOfTableTypes() || index > instance->m_tables[elem->tableIndex()]->size() || index + elem->functionIndex().size() > instance->m_tables[elem->tableIndex()]->size())) {
```

Comment:
Why did you change this `>=` operator as to '>' ?

Reply:
In some cases it causes error (to be more exact, in tests: **binary.wast**, **binary-leb128.wast** and **elem.wast**), when it tries to write nothing to a table that has no size.

Here (from file **elem.wast:174-177**):
```
(module
  (table 0 funcref)
  (elem (i32.const 0))
)
```

Although it didn't cause assert failure, it didn't work propely, I think. I noticed it when I started to print out semantic errors.